### PR TITLE
travis: Build and publish NuGets on tags

### PR DESCRIPTION
**Pull request by hbons**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -34,18 +34,18 @@ sdk: nuget-download
 	mono src/.nuget/nuget.exe pack FigmaSharp.nuspec
 
 package: check-dependencies
-	msbuild FigmaSharp/FigmaSharp/FigmaSharp.csproj /p:Configuration=Release /t:Restore
-	msbuild FigmaSharp/FigmaSharp.Cocoa/FigmaSharp.Cocoa.csproj /p:Configuration=Release /t:Restore
-	msbuild FigmaSharp/FigmaSharp.Forms/FigmaSharp.Forms.csproj /p:Configuration=Release /t:Restore
+	msbuild FigmaSharp/FigmaSharp/FigmaSharp.csproj /p:Configuration=Release /restore
```
Comment (netonjm) [inline Makefile]: why?

File: Makefile [hunk 2]
```diff
@@ -12,10 +12,10 @@ clean:
 	find . -type d -name packages -exec rm -rf {} \;
 
 pack:
-	msbuild FigmaSharp.Views.sln $(ARGS) /p:CreatePackage=true
+	msbuild FigmaSharp.Views.sln $(ARGS) /p:CreatePackage=true /restore
```
Comment (netonjm) [inline Makefile]: This needs to be changed to use the FigmaSharp.sln

File: Makefile [hunk 3]
```diff
@@ -12,10 +12,10 @@ clean:
...
 
 install:
-	msbuild FigmaSharp.Views.sln $(ARGS) /p:InstallAddin=true
+	msbuild FigmaSharp.Views.sln $(ARGS) /p:InstallAddin=true /restore
```
Comment (netonjm) [inline Makefile]: This needs to be changed to use the FigmaSharp.sln

File: Makefile (hunk 1 above)
Comment (hbons) [reply Makefile]: /t:Restore doesn't always build. /restore does.

Comment (netonjm) [reply Makefile]: just curious, in what case is not building?

Comment (hbons) [reply Makefile]: Not sure. Once I changed it it started working. :)

**Pull request merged**